aio/aio-proxy/aio_proxy/request/search_params_builder.py

```diff
@@ -190,3 +191,12 @@ def check_and_validate_params(request, params):
         # Prevent performance issues by refusing query terms less than 3 characters
         # unless another param is provided
         check_short_terms_and_no_param(params)
+
+    @staticmethod
```

Comment:
Why do you use a static method here ? Will you access it outside of a class instance ?

Reply:
It's so I don't have to initiate an object here `search_params = SearchParamsBuilder.extract_params(request, search_type)`